proto/api.proto

```diff
@@ -288,6 +288,17 @@ service Api {
     };
   }
 
+  rpc PostMultiRouteTradeSwap(MultiRouteTradeSwapRequest) returns (MultiRouteTradeSwapResponse) {
```

Comment:
naming: I don't think `Multi` should be part of this name. This is a swap with a route specification, and it's valid if the user decides to give a route specification of size 1. So I think `PostRouteTradeSwap` or `PostRouteSwap` would be better